src/components/FreakTile/__test__/FreakTile.test.jsx

```diff
@@ -4,15 +4,27 @@ import FreakTile from '../FreakTile';
 
 describe('FreakTile', () => {
   it('should render the freak tile', () => {
-    render(<FreakTile id={ 5 } picture="some-url" name="naruto" />);
+    render(<FreakTile id={ 5 } picture="some-url" name="name" />);
```

Comment:
There is a best practice for mock data, use realistic values, give this a name you expect to see in the application, the URL is probably not worth making realistic. And for the ID, try 1 instead of 5, 1 is the first number, it seems natural to see as a mock, 5 has nothing special about it, if someone reads a test they might start wondering why you chose 5 specifically.